src/swsssdk/port_util.py

```diff
@@ -11,12 +11,14 @@
 in multi-asic platform.
 """
 SONIC_ETHERNET_BP_RE_PATTERN = "^Ethernet-BP(\d+)$"
+SONIC_VLAN_RE_PATTERN = "^Vlan(\d+)$"
 SONIC_PORTCHANNEL_RE_PATTERN = "^PortChannel(\d+)$"
 SONIC_MGMT_PORT_RE_PATTERN = "^eth(\d+)$"
 
 
 class BaseIdx:
     ethernet_base_idx = 1
+    vlan_interface_base_idx = 2000
```

Comment:
@prsunny @judyjoseph do you think it leaves enough indexes to portchannels in multiple-ASIC case?

Reply:
I feel 1000 port channel interfaces should be good enough even for Chassis. But I am looking to see why the vlan_interface_base_idx is defined  .. is it for this API get_vlan_interface_oid_map() ?

Reply:
I think this just simply leaves a gap from 6000-9000 after `2000+4094`, why not start from 4000?

Reply:
2K BaseIdx is what our Arch and Guohan agreed on. If there is any concern about the size of pool, should discuss with them.

Reply:
@judyjoseph 
>  is it for this API get_vlan_interface_oid_map() ?
Theoretically index can be used anywhere where a numeric identification for interface is needed. 
In my case I am using it for interface ID in the SNMP Interface MIB (RFC1213).

Reply:
lgtm .. I see that the index range discussion already happened.